### PR TITLE
Various changes, please see description below

### PR DESCRIPTION
**Pull request by kristina-voit**
Please see user stories #7 , #8 , #11

- added toggle function on dashboard medicine card
- added "Hersteller" to medicine
- added "required" for input field on dashboard to-do card
- fixed link for phonecall and emails in appointment card
- started working on new medicine form
- deleted unneccessary files

### CODE REVIEW

File: src/stories/Introduction.stories.mdx [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import { Meta } from '@storybook/addon-docs';
-import Code from './assets/code-brackets.svg';
-import Colors from './assets/colors.svg';
-import Comments from './assets/comments.svg';
-import Direction from './assets/direction.svg';
-import Flow from './assets/flow.svg';
-import Plugin from './assets/plugin.svg';
-import Repo from './assets/repo.svg';
-import StackAlt from './assets/stackalt.svg';
+import { Meta } from "@storybook/addon-docs";
```
Comment (JessicaLoers) [inline src/stories/Introduction.stories.mdx:1]: Do we need this Intoduction Document for Storybook?

File: src/stories/Page.jsx [hunk 2]
```diff
@@ -1,51 +1,73 @@
...
           <strong>component-driven</strong>
-        </a>{' '}
+        </a>{" "}
         process starting with atomic components and ending with pages.
       </p>
       <p>
-        Render pages with mock data. This makes it easy to build and review page states without
-        needing to navigate to them in your app. Here are some handy patterns for managing page data
-        in Storybook:
+        Render pages with mock data. This makes it easy to build and review page
```
Comment (JessicaLoers) [inline src/stories/Page.jsx:30]: And again... ;) if we don't need this, you could delete this.

**Review by JessicaLoers — APPROVED** [review]
I have looked at the changes - well done!
Unfortunately this link resulted in a 404.

This link throws a 404: https://vercel.com/kristina-voit/capstone-project/BDNiSpWGjwA4GxuinEFCXnaRWe4r

File: src/stories/Introduction.stories.mdx (hunk 1 above)
Comment (kristina-voit) [reply src/stories/Introduction.stories.mdx:1]: Didn't do any work on the storybook yet. Will leave it in until I start working on that. :-)

File: src/stories/Page.jsx (hunk 2 above)
Comment (kristina-voit) [reply src/stories/Page.jsx:30]: Same here - will delete it later!

**Pull request merged**